server/public/styles.css

```diff
@@ -450,3 +465,85 @@ i:hover {
   border: 2px solid var(--dark-color-b);
   object-fit: cover;
 }
+
+.login {
+  background-color: var(--dark-color-a);
+  width: 90%;
+  border-radius: 3rem;
+  color: white;
+  padding: 3rem;
+  box-shadow: 9px 9px 20px 1px rgb(0 0 0 / 88%);
+}
+
+.login input,
+.extra-user-inputs {
+  color: var(--dark-color-a);
+  width: 100%;
+  border-radius: 3rem;
+  height: 2rem;
+  margin-bottom: 1rem;
+  letter-spacing: 0.25rem;
+  font-style: 1.5rem;
+  padding-left: 1rem;
+  font-size: 1.5rem;
+  overflow: hidden;
+}
+
+.register {
+  margin-bottom: 2rem;
+}
+
+.create-username h2,
+.create-password h2 {
+  margin: 0.5rem;
+}
+
+.register a {
+  text-decoration: none;
+  color: white;
+}
+
+.register-on {
+  color: var(--highlight);
+}
+
+.upload-image {
+  width: 10rem;
+  height: 10rem;
+  background-color: grey;
+  border-radius: 50%;
+}
+
+.registration-photo {
+  width: 10rem;
+  height: 10rem;
+  border-radius: 50%;
+  object-fit: cover;
+}
+
+.image-upload2 {
+  display: flex;
+  justify-content: center;
+  width: 10rem;
+  height: 10rem;
+  background-color: var(--dark-color-b);
+  border-radius: 50%;
+}
+
+.image-upload2 input[type="file"]::-webkit-file-upload-button {
```

Comment:
Instead of 2, how about `.profile-image-upload`? Just sticking a number on the end isn't very helpful to the reader of the code.